src/Core/Form/IdentifiableObject/DataProvider/ProductFormDataProvider.php

```diff
@@ -124,7 +125,7 @@ public function getDefaultData()
             'header' => [
                 'type' => ProductType::TYPE_STANDARD,
             ],
-            'basic' => [
+            'description' => [
```

Comment:
What are default data for `specifications` ?

Reply:
No default data, they are empty like most of the data